packages/veritone-react-common/src/components/OCREngineOutputView/OCRObject/index.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { number, string, func } from 'prop-types';
+import cx from 'classnames';
+
+import { msToReadableString } from 'helpers/time';
+import styles from './styles.scss';
+
+class OCRObject extends Component {
```

Comment:
i'm not convinced this needs to be a standalone component. There is only 4 elements being rendered and no isolated logic specific to this component. I'd lean towards to putting this in `OCREngineOutputView` as I don't feel we're gaining anything but additional bloat.